config/rca.conf

```diff
@@ -65,6 +65,14 @@
     "shard-request-cache-rca": {
       "shard-request-cache-threshold" : 0.9
     },
+    "search-back-pressure-rca": {
+       "max-heap-usage-increase": 80,
```

Comment:
I am guessing this is kept at 80 considering node duress heap setting is set at 70% here - https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/search/backpressure/settings/NodeDuressSettings.java#L24C37-L24C51?

Considering this is dependent on above value, is there a way to extract above value and use it to define these settings? 
For example, what happens if I change HEAP_THRESHOLD to 85% at OpenSearch level?

Reply:
Let me see, How to best update these values with cluster's current set value.

Reply:
Done